master/networking/migrate-pools.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Migrate from one IP pool to another
+---
+
+### Big picture
+
+Migrate pods from one IP pool to another on a running cluster without network disruption.
+
+### Value
+
+Pods are assigned IP addresses from IP pools that you configure in {{site.prodname}}. As pods increase, you may want pods to move a larger CIDR. Or, you may need to move pods from a CIDR that was used by mistake. {{site.prodname}} lets you migrate from one IP pool to another one on a running cluster without network disruption. 
```

Comment:
"As the number of pods increases" rather than "As pods increase"

> you may want pods to move a larger CIDR

I'd instead recommend saying "you may need to increase the number of addresses available for pods to use"

Reply:
"increase or decrease" perhaps?

Reply:
Fixed.

Reply:
Fixed.